tests/perf-test/OptionsSelector.perf-test.tsx

```diff
@@ -35,8 +38,11 @@ jest.mock('../../src/components/withNavigationFocus', () => (Component) => {
     return WithNavigationFocus;
 });
 
-const generateSections = (sectionConfigs) =>
-    _.map(sectionConfigs, ({numItems, indexOffset, shouldShow = true}) => ({
+type GenerateSectionsProps = Array<{numItems: number; indexOffset: number; shouldShow?: boolean}>;
```

Comment:
NAB, but we should avoid abbreviations like `num`. An alternative to `numberOfItems` is `itemCount`
```suggestion
type GenerateSectionsProps = Array<{numberOfItems: number; indexOffset: number; shouldShow?: boolean}>;
```